CHANGELOG.md

```diff
@@ -20,6 +47,10 @@ Versioning](https://semver.org/spec/v2.0.0.html).
   resources for the `gitopsservices.pipelines.openshift.io` API group. This is
   necessary to automate checks that ensure the cluster is using GitOps operator.
 
+- Added necessary permessions for api-resource-collector so that the new rule
+  `cluster_logging_operator_exist` can be evaluated properly. [1]
+  [1] https://github.com/ComplianceAsCode/content/pull/8511
+
 ### Fixes
 
 - The compliance content images have moved to
```

Comment:
I think we should update the Customer Case below so that it references a public bugzilla or an issue in this new repository.